model/service_mgmt/v1/cluster_parameters_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This represents the parameters needed by Managed Service to create a cluster.
+struct ClusterParameters {
+	Name String
+	DisplayName String
+	link Region CloudRegion
```

Comment:
If this is a link where is in the target of that link? I don't see in this API definition any method that returns those regions. Is this intended to be a link to some other service?